package.json

```diff
@@ -11,6 +11,9 @@
     "start:federate": "webpack serve --host localhost --port 9006 --config webpack.dev.js",
     "test": "jest --verbose",
     "test:watch": "jest --watch --verbose",
+    "cypress:open:it": "cypress open --project cypress --config-file cypress.it.json",
+    "cypress:run:it": "cypress run --project cypress --config-file cypress.it.json",
+    "test:it": "start-server-and-test 'start:mocked' http://localhost:8080 'cypress:run:it' && echo",
```

Comment:
what is the `echo` used for please?

Reply:
@jomarko see the description of this commit https://github.com/kiegroup/kogito-apps/pull/1092/commits/0d291659b3ab0aec3c3d8893a6c3eca88296ec7d